test/unit/spec/VerifyWebauthn_spec.js

```diff
@@ -149,47 +212,168 @@ function (Okta,
     expect(test.form.titleText()).toBe(desiredTitle);
   }
 
-  Expect.describe('Webauthn Factor', function () {
+  function testWebauthnFactor (setupFn, webauthnOnly) {
     itp('shows the right beacon and title for webauthn', function () {
-      return setupWebauthnFactor({webauthnSupported: true}).then(function (test) {
+      return setupFn({webauthnSupported: true}).then(function (test) {
         expectHasRightBeaconImage(test, 'mfa-webauthn');
         expectTitleToBe(test, 'Security Key or Built-in Authenticator');
       });
     });
 
     itp('shows error if browser does not support webauthn', function () {
-      return setupWebauthnFactor({webauthnSupported: false}).then(function (test) {
+      return setupFn({webauthnSupported: false}).then(function (test) {
         expect(test.form.el('o-form-error-html')).toHaveLength(1);
+        var errorMessage = webauthnOnly ?
+          'Security key or built-in authenticator is not supported on this browser. ' +
+          'Contact your admin for assistance.' :
+          'Security key or built-in authenticator is not supported on this browser. ' +
+          'Select another factor or contact your admin for assistance.';
         expect(test.form.el('o-form-error-html').find('strong').html())
-          .toEqual('Security key or built-in authenticator is not supported on this browser. ' +
-          'Select another factor or contact your admin for assistance.');
+          .toEqual(errorMessage);
       });
     });
 
     itp('does not show error if browser supports webauthn', function () {
-      return setupWebauthnFactor({webauthnSupported: true}).then(function (test) {
+      return setupFn({webauthnSupported: true}).then(function (test) {
         expect(test.form.el('o-form-error-html')).toHaveLength(0);
       });
     });
 
     itp('shows a spinner while waiting for webauthn challenge', function () {
-      return setupWebauthnFactor({webauthnSupported: true}).then(function (test) {
+      return setupFn({webauthnSupported: true}).then(function (test) {
         expect(test.form.el('webauthn-waiting').length).toBe(1);
+        return Expect.waitForSpyCall(webauthn.isNewApiAvailable);
```

Comment:
you don't need this `Except` right? the test ends here.

Reply:
You're right sorry! Must've added by mistake when refactoring! Thanks for catching this!!!